### DISCUSSION TITLE
Remove the cancel delay

### DISCUSSION THREAD

**Discussion by mathmaj** [Keriew/augustus#291] [Ideas]
Hi! 
thanks a lot for the very nice work! I love it.

If the game is paused or 10% speed, and I want to cancel an action, there is a big delay.
Do you think it would be possible to remove it?

**Comment by crudelios** [comment]
Hi, I'm not entirely sure it's possible to remove it, but I can have a look.

Due to the way the game engine works, undoing is already very bug-prone. I think making it instantaneous will probably not allow the game to update some internal stuff, resulting in a borked city. I'll eventually have a look at this though, as it would certainly make the game feel more modern!

**Reply by mathmaj** [reply]
Thanks for your reply!
